test/c/test_circuit.c

```diff
@@ -73,7 +73,7 @@ int test_circuit_copy(void) {
     QkCircuit *copy = qk_circuit_copy(qc);
     for (int i = 0; i < 10; i++) {
         qk_circuit_measure(qc, i, i);
-        int qubits[1] = {
+        uint32_t qubits[1] = {
             i,
         };
```

Comment:
Strictly we should probably change the type of the iteration variable too, although the behaviour of this is at least well-defined and cannot cause problems given the hard-coded loop bounds.

(We do this a bunch, so it's possibly not worth bothering right now.)